data/sykmeldinger.ts

```diff
@@ -1,21 +1,26 @@
-import { avbruttSykmelding } from "./sykmeldinger/avbruttSykmelding";
```

Comment:
Her eksponeres kun de ferdige sykmeldingen, ikke de nye, feks "ny-sykmelding-variasjon-1-sykmelding". Sykefravær som blir sendt til frontend inneholder derfor et tomt array for sykmeldinger.

Reply:
woops, det var en feil. skal fikse